ner_v2/detectors/numeral/utils.py

```diff
@@ -81,17 +97,28 @@ def get_number_from_number_word(text, number_word_dict):
                 result = current = 0
                 result_text, current_text = '', ''
 
+            if digit_len > prev_digit_len and on_number and prev_scale == scale:
+                current = current * (10 ** digit_len)
+
+            span = re.search(word, spanned_text).span()
+            start_span = end_span + span[0]
+            end_span += span[1]
+            spanned_text = spanned_text[span[1]:]
```

Comment:
F841 local variable 'spanned_text' is assigned to but never used
F821 undefined name 'spanned_text'